shell-completion/bash/systemd-run

```diff
@@ -25,10 +25,30 @@ __systemctl() {
 __get_slice_units () { __systemctl $1 list-units --all -t slice \
                            | { while read -r a b c d; do echo " $a"; done; }; }
 
+__get_properties () {
+    systemd-analyze transient-settings scope service 2>/dev/null |
+        while read -r p; do [ -z "$p" ] || echo "$p="; done
+}
+
+__get_path_properties () {
+    systemd-analyze transient-settings path |
```

Comment:
Missing `2>/dev/null` ?

Reply:
Yeah, maybe.

Reply:
Yeah. `systemd-analyze` might not be installed. It's better to offer no completions than to spam the console with error messages.

Reply:
#38210